website/pages/docs/commands/connect/envoy.mdx

```diff
@@ -196,6 +196,15 @@ $ consul connect envoy -gateway=terminating -register \
   -address '{{ GetInterfaceIP "eth0" }}:8443'
 ```
 
+### Ingress Gateways
+
+The terminating gateway Envoy process can be started with.
```

Comment:
```suggestion
The terminating gateway Envoy process can be started with the following command.
```

Reply:
The text should reference ingress gateways rather than terminating

Reply:
updated with correct verbage and grammar.